### DISCUSSION TITLE
For Faster-Whisper-XXL, are we supposed to download the full > 1 GB file every time there's an update?

### DISCUSSION THREAD

**Discussion by nhan000** [Purfview/whisper-standalone-win#363] [Q&A] [answered]
For Faster-Whisper, the update size is usually ~ 100 MB, which only has the (I assumed) updated components and I just need to replace the old ones. I'm wondering if there's a quicker/lighter way to update Faster-Whisper-XXL?

**Comment by Purfview** [comment, marked as answer]
>For Faster-Whisper-XXL, are we supposed to download the full > 1 GB file every time there's an update?

Yes.

>For Faster-Whisper, the update size is usually ~ 100 MB, which only has the (I assumed) updated components and I just need to replace the old ones.

Those are/were not updates per se, but whole updated program too as "XXL". It was smaller because it didn't had "XXL" features and cuBLAS + cuDNN.

>I'm wondering if there's a quicker/lighter way to update Faster-Whisper-XXL?

There could be if I would provide just an incremental patch.
